### PR TITLE
Miner -> staker in contracts

### PR DESCRIPTION
**Pull request by szotov**
Based on #949 

Renames miner to staker in solidity contracts and in tests for them.

Refs #800

### CODE REVIEW

File: tests/blockchain/eth/contracts/main/staking_escrow/test_tracking.py [hunk 1]
```diff
@@ -158,12 +159,12 @@ def test_pre_deposit(testerchain, token, escrow_contract):
...
     testerchain.wait_for_receipt(tx)
 
-    # Deposit tokens for 1 miner
+    # Deposit tokens for 1 staker
     owner = testerchain.interface.w3.eth.accounts[1]
     tx = escrow.functions.preDeposit([owner], [1000], [10]).transact({'from': creator})
     testerchain.wait_for_receipt(tx)
     assert 1000 == token.functions.balanceOf(escrow.address).call()
-    assert 1000 == escrow.functions.minerInfo(owner).call()[VALUE_FIELD]
+    assert 1000 == escrow.functions.getAllTokens(owner).call()
```
Comment (KPrasch) [inline tests/blockchain/eth/contracts/main/staking_escrow/test_tracking.py]: This is not a rename - Is it an Intentional change?

Comment (KPrasch) [reply tests/blockchain/eth/contracts/main/staking_escrow/test_tracking.py]: Whoops! Nevermind.

File: docs/source/architecture/contracts.md [hunk 2]
```diff
@@ -59,59 +59,59 @@ Alice can refund ETH for any inactive periods without revoking the policy by usi
 
 ### Ursula Locks Tokens
 
-In order to become a participant of the network, a miner stakes tokens in the `MinersEscrow` contract.
-The miner allows the (mining) contract to perform a transaction using the `NuCypherToken.approve(address, uint256)` method
+In order to become a participant of the network, a staker stakes tokens in the `StakingEscrow` contract.
+The staker allows the (mining) contract to perform a transaction using the `NuCypherToken.approve(address, uint256)` method
```
[the remark this answers is not in the document]
Comment (szotov) [reply docs/source/architecture/contracts.md]: Oops, missed this

File: docs/source/architecture/contracts.md [hunk 3]
```diff
@@ -59,59 +59,59 @@ Alice can refund ETH for any inactive periods without revoking the policy by usi
...
 Also, the reward depends on the number of periods during which the tokens will be locked: if the tokens will be locked for half a year, the coefficient is 1.5.
 The minimum coefficient is 1 (when tokens will get unlocked in the next period), and the maximum is 2 (when the time is 1 year or more).
 The reward is calculated separately for each stake that is active during the mining period and all rewards are summed up.
-The order of calling `MinersEscrow.mint()` by miners (e.g. who calls first, second etc) doesn't matter.
-Miners can claim their rewards by using the `MinersEscrow.withdraw(uint256)` method. Only non-locked tokens can be withdrawn.
+The order of calling `StakingEscrow.mint()` by stakers (e.g. who calls first, second etc) doesn't matter.
+Stakers can claim their rewards by using the `StakingEscrow.withdraw(uint256)` method. Only non-locked tokens can be withdrawn.
 
 
 ### Ursula Generates Policy Rewards
```
[the remark this answers is not in the document]
Comment (szotov) [reply docs/source/architecture/contracts.md]: Hmm, I thought fee for policy and reward for staking

**Pull request merged**